global/test/TestCommandLineArgumentsTutorial.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef TESTCOMMANDLINEARGUMENTSTUTORIAL_HPP_
```

Comment:
Needs the copyright notice (this is causing a [test to fail](https://github.com/Chaste/Chaste/actions/runs/9385768547/job/25844727324?pr=282)). Should be a matter of either copying and pasting from another test or running the script mentioned in the failure message.

Reply:
Corrected in [c005edb](https://github.com/Chaste/Chaste/pull/282/commits/c005edbb21e3d856622fbebe6951aa9f8fdccd51).